test/index.spec.js

```diff
@@ -0,0 +1,212 @@
+const waitForActions = require('../');
+const configureStore = require('redux-mock-store').default;
+const thunkMiddleware = require('redux-thunk').default;
+
+const initialState = {};
+const action1 = { type: 'ACTION-1', payload: { id: 1, name: 'ACTION ONE' } };
+const action2 = { type: 'ACTION-2', payload: { id: 2, name: 'ACTION TWO' } };
+const action3 = { type: 'ACTION-3', payload: { id: 3, name: 'ACTION THREE' } };
+const action4 = { type: 'ACTION-4', payload: { id: 4, name: 'ACTION FOUR' } };
+const timeoutError = { code: 'ETIMEDOUT', name: 'TimeoutError' };
+const cancelledError = { code: 'ECANCELLED', name: 'CancelledError' };
+const asyncActionsCreator = (actions, timeoutMs = 10) => (dispatch) => {
```

Comment:
could we rename `timeoutMs` to `timeout`? Also the `timeout` from setTimeout can be renamed to `timeoutId`.